### DISCUSSION TITLE
How to get settled amount into your bank account

### DISCUSSION THREAD

**Discussion by scmishra-cse** [community/community#35535] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/84576053/194717232-344cfb15-30d0-4019-91ef-6e16b7c95a2b.png)
This is my express account home screen. How can I redeem my money in a bank account which I have provided?

**Comment by kraanzu** [comment]
Hey @scmishra-cse, What's the conversion rate for you i.e USD to INR? 
Mine is very odd and Github Support isn't responding

**Comment by jlord** [comment, marked as answer]
Hiya! There is a 60 day probation period for all new accounts ([details here](https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing)), after which payouts into bank accounts will be automatic on the 22nd of each month for most regions.

**Reply by Palanikannan1437** [reply]
Hey @jlord, it has been over 90 days (payment was initialised on 9th Sep) and I've not received anything in my bank account yet, could you please look into this and clarify? Thank you!😄

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/73993394/214498511-03a311c7-94d9-47d3-b3aa-2187f1311e90.png">

**Reply by jlord** [reply]
Hi @Palanikannan1437 :wave: could you [start a support ticket here](https://support.github.com/contact/account) (select GitHub Sponsors) we'll get you sorted but because these requests involve personal/banking information it's best to handle there. Thank you!

**Comment by 2958888238** [comment]
I want to withdraw money from a bank check.
![Screenshot_20250602_183242_Bitget](https://github.com/user-attachments/assets/03f57a5f-4bad-47ea-bd84-5d0a19669909)
![Screenshot_20250602_183218_Bitget](https://github.com/user-attachments/assets/396aab8d-6d83-40eb-a18b-b0df965e5c73)

**Comment by pichay429-pixel** [comment]
Wingbank     102968788
![IMG_1763714717286](https://github.com/user-attachments/assets/ecdf6d89-fba8-479d-9f82-617939061ec2)

**Reply by aza773072700-del** [reply]
Bashir Fadl Muhammad Ibrahim Haddad I need $3000. My phone number is +967773072700

**Reply by aza773072700-del** [reply]
🇾🇪Yemen

**Comment by 2958888238** [comment]
มิฉาชีพติดต่อกับด่วนครับ0614381552

ในวันที่ อา. 23 พ.ย. 2025 01:19 pichay429-pixel ***@***.***>
เขียนว่า:

**Comment by 2958888238** [comment]
มิฉาชีพแอบเอาเงินแล้วบังธนาคารไว้ไม่ให้เห็นยอดเงินครับ

ในวันที่ อา. 23 พ.ย. 2025 01:56 กา แฟ ***@***.***> เขียนว่า:

**Comment by aza773072700-del** [comment]
I need $10,000 Name: Bashir Fadl Muhammad Ibrahim Haddad My phone number is +967773072700 🇾🇪Yemen

**Comment by aza773072700-del** [comment]
5000دولار
بشير فضل محمد ابراهيم حداد

+967773072700
اليمن 🇾🇪

في الجمعة، ٢٦ يونيو ٢٠٢٦، ٨:٥٩ م ramzan1986520-alt ***@***.***>
كتب:

**Comment by aza773072700-del** [comment]
بشير فضل محمد ابراهيم حداد

في السبت، ٢٧ يونيو ٢٠٢٦، ١٢:٤١ ص بشير الحداد ***@***.***> كتب:

**Comment by 2958888238** [comment]
โอนเข้าธนาคาร

ในวันที่ จ. 20 ก.ค. 2026 09:25 aza773072700-del ***@***.***>
เขียนว่า:
